### DISCUSSION TITLE
Capacitor Camera on iOS 14 iPad

### DISCUSSION THREAD

**Discussion by juancaceres** [ionic-team/capacitor#3576] [Q&A]
We are having problems with capacitor on ios 14 with the camera, somebody knows about some issue? Know problem? we are using capacitor 2.4

Thanks in advance if somebody can help...

**Comment by dotNetkow** [comment]
What is the specific issue? What error messages do you get? Can you share the code you're using?

**Reply by juancaceres** [reply]
**Issue**: The preview in the camera get freeze, just happend on ipads with iOS 14.
**Error**: There's not a forma error in the log or output, just get freeze the camera.
**The Code**:
import { Plugins, CameraResultType, CameraSource, CameraDirection } from '@capacitor/core';
const { Camera } = Plugins;
try {
      const imageP = await Camera.getPhoto({
        correctOrientation: true,
        height: 50,
        width: 50,
        quality: 50,
        saveToGallery: false,
        allowEditing: false,
        direction : CameraDirection.Rear,
        source: CameraSource.Photos,
        resultType: CameraResultType.DataUrl,
      }).catch((e) => {
        throw new Error(e);
      });
      //console.log(imageP.dataUrl );
      this.loadingC.loadingPresent('Person recognition');
      this.urlPhoto = imageP.dataUrl;
      this.flagType = 'aws';
      this.profileUrl =  this.domSanitizer.bypassSecurityTrustResourceUrl(imageP && (imageP.dataUrl));
      this.uploadImage(this.urlPhoto , '', 'large', this.tokenObj.main_token, true);
    } catch (e) {
      //console.log('Photo Cancelled');
    }

**Comment by hansenchristian** [comment]
I'm running into the same Problem
